ReactAndroid/src/main/java/com/facebook/react/bridge/JSBundleLoader.java

```diff
@@ -31,7 +31,7 @@ public void loadScript(ReactBridge bridge) {
         if (fileName.startsWith("assets://")) {
           bridge.loadScriptFromAssets(context.getAssets(), fileName.replaceFirst("assets://", ""));
```

Comment:
Nope...turns out that function takes an "assetName" param. So we're gonna leave it alone here, and change it where the script is executed in `OnLoad.cpp`